frontend/packages/core/src/stories/clipboard-button.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import type { Meta } from "@storybook/react";
+
+import type { ClipboardButtonProps } from "../button";
+import { ClipboardButton } from "../button";
+
+export default {
+  title: "Core/Clipboard Button",
```

Comment:
what do you think of
```suggestion
  title: "Core/Buttons/Clipboard",
```

Reply:
in this PR, going to move the other button stories under `Core/Buttons/` as well

Reply:
![Screen Shot 2020-11-04 at 4 53 51 PM](https://user-images.githubusercontent.com/39421794/98171822-5db9a880-1ebe-11eb-80e4-63025461a4ca.png)